Phaethon/WebClient/Views/Shared/_Layout.cshtml

```diff
@@ -1,4 +1,19 @@
 ﻿@using LanguagePack = WebClient.Resources.Language_Files.LanguagePack;
+@{
+
+    var viewControlerName = ViewContext.RouteData.Values["controller"].ToString();
+    if (viewControlerName != "Login")
+    {
+        if (Session["ID"] == null)
```

Comment:
both of thease if stmts could be added to same if, so less space is taken and more readable, just add && between them :)